src/tasks/processIssueComment/probot-processIssueComment.js

```diff
@@ -24,6 +24,13 @@ async function processAddContributor({
     contributions,
     branchName,
 }) {
+    if (contributions.length === 0) {
```

Comment:
@Berkmann18 this code looks good to me. If we can fix the PR commit/merge issues then good to merge! :) 🎉

Reply:
Resolved, woo 🎉, go github